main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"runtime"
+)
+
+// #include <unistd.h>
+import "C"
+
+const (
+	listenSocket = ":6666"
+
+	stressBinary          = "stress"
+	stressTimeout         = 60
+	stressIterations      = 5
+	stressDefaultMemoryMB = 256
+	stressMemoryHangSec   = 2
+)
+
+// Common variables.
+var (
+	description = "Simple stress checker for Kubernetes nodes."
+	gitCommit   = "n/a"
+	name        = "kube-stresscheck"
+	source      = "https://github.com/giantswarm/kube-stresscheck"
+)
+
+func main() {
+	// Print version.
+	if (len(os.Args) > 1) && (os.Args[1] == "version") {
+		fmt.Printf("Description:    %s\n", description)
+		fmt.Printf("Git Commit:     %s\n", gitCommit)
+		fmt.Printf("Go Version:     %s\n", runtime.Version())
+		fmt.Printf("Name:           %s\n", name)
+		fmt.Printf("OS / Arch:      %s / %s\n", runtime.GOOS, runtime.GOARCH)
+		fmt.Printf("Source:         %s\n", source)
+		return
+	}
+
+	var help bool
+
+	flag.BoolVar(&help, "help", false, "Print usage and exit")
```

Comment:
You can do something like `os.Args[1] == "--help"`, which might make this a bit tidier